Runtimes/Supplemental/Synchronization/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+cmake_minimum_required(VERSION 3.29)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE YES) # ToDo: What does this do?
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../cmake/modules")
+
+if($ENV{BUILD_NUMBER})
+  math(EXPR BUILD_NUMBER "$ENV{BUILD_NUMBER} % 65535")
+  set(BUILD_NUMBER ".${BUILD_NUMBER}")
+endif()
+project(SwiftSynchronization
+  LANGUAGES Swift
+  VERSION 6.1.0${BUILD_NUMBER})
+
+include(GNUInstallDirs)
+
+if(NOT PROJECT_IS_TOP_LEVEL)
+  message(SEND_ERROR "Swift Synchronization must build as a standalone project")
+endif()
+
+set(${PROJECT_NAME}_SWIFTC_SOURCE_DIR
+  "${PROJECT_SOURCE_DIR}/../../../"
+  CACHE FILEPATH "Path to the root source directory of the Swift compiler")
+
+find_package(SwiftCore)
+include(gyb)
+include(AvailabilityMacros)
+
+add_compile_options(
+  $<$<COMPILE_LANGUAGE:Swift>:-explicit-module-build>
+  $<$<COMPILE_LANGUAGE:Swift>:-enable-builtin-module> # Are these contradictory?
```

Comment:
These don't conflict. explicit module builds affect how the driver looks up modules, while `builtin-module` allows libraries that are not part of the core `Swift` module to use builtins. It's not for use outside of the distribution of standard libraries, but everything in Supplemental is part of that distribution so it's fine.